plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/aggregation/ImplementAvg.java

```diff
@@ -58,6 +59,7 @@
     public Optional<AggregateExpression> rewrite(AggregateFunction aggregateFunction, Captures captures, RewriteContext context)
     {
         Variable argument = captures.get(ARGUMENT);
-        return Optional.of(new AggregateExpression(aggregateFunction.getFunctionName(), context.getIdentifierQuote().apply(argument.getName()), true));
+        PinotColumnHandle columnHandle = (PinotColumnHandle) context.getAssignment(argument.getName());
+        return Optional.of(new AggregateExpression(aggregateFunction.getFunctionName(), context.getIdentifierQuote().apply(columnHandle.getColumnName()), true));
```

Comment:
`context.getIdentifierQuote()` is identity, right? is it OK?

Reply:
It's ok: the DynamicTablePqlExtractor quotes identifiers.